web/src/app/Login/login.Component.ts

```diff
@@ -1,14 +1,44 @@
-import {Component, OnInit} from '@angular/core';
+import { Component, OnInit } from '@angular/core';
+import { isNullOrUndefined } from 'util';
+import { LoginService } from './shared/login.service';
+import { UserLoginModel } from '.././services/swagger-generated/models';
 
 @Component({
   selector: 'login',
   templateUrl: 'login.component.html'
 })
 
 export class LoginComponent implements OnInit {
-  constructor() {
+
+  userEmail: string;
+  userPassword: string;
+  loginModel: UserLoginModel;
+  invalidCredentials : boolean;
```

Comment:
why this invalidCredential variable is used, remove unnecessary variables.

Reply:
Done